src/devices/Bmx280/samples/Bme280.sample.cs

```diff
@@ -23,53 +23,68 @@ static async Task Main(string[] args)
             //set this to the current sea level pressure in the area for correct altitude readings
             const double defaultSeaLevelPressure = 1033.00;
 
-            var i2cSettings = new I2cConnectionSettings(busId, Bme280.DefaultI2cAddress);
+            var i2cSettings = new I2cConnectionSettings(busId, Bme280.DefaultI2cAddress + 1);
```

Comment:
I suppose you don't need this +1 any longer now that you've fixed the address?

Reply:
Oops, fixed!